monitoring/index.md

```diff
@@ -15,6 +15,10 @@ Use this data to understand [which message handlers need to be optimized](/tutor
 
 ## Endpoint health
 
-ServiceControl can track which endpoint instance are running by listening to a [continual stream of heartbeat messages](/nservicebus/operations/heartbeat.md). Endpoints that fail to send heartbeats within a defined period of time are considered to be offline and an [alert is raised](/servicecontrol/contracts.md) to an operator.
+A ServiceControl instance can track which endpoint instance are running by listening to a [continual stream of heartbeat messages](/monitoring/heartbeats/). Endpoints that fail to send heartbeats within a defined period of time are considered to be inactive. When an inactive endpoint starts sending heartbeats again, it becomes active. 
```

Comment:
Add headings? ### heartbeats and ### custom checks?

Reply:
It's a bit hard here because we first talk about what the endpoint does, then what ServiceControl does, then ServicePulse, then integration events.